pkg/cloud/services/elb/loadbalancer_test.go

```diff
@@ -1847,6 +1847,70 @@ func TestReconcileV2LB(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "ensure NLB without SGs doesn't attempt to add new SGs",
```

Comment:
This test is invalid, as it's passing even when the conditions aren't satisfied. It needs to have some more mocks fleshed out in order to be accurate.

/hold

Reply:
/unhold